app/controllers/invitations_controller.rb

```diff
@@ -0,0 +1,35 @@
+class InvitationsController < ApplicationController
+  before_action :authenticate!
+  before_action :set_user!
+  before_action :set_invitation!
+
+  def show
+    redirect_to_organization! if @user.student_of? @organization
```

Comment:
But, the link is `http://mumuki.io/join/1f3g-curso` and not `http://orga.mumuki.io/join/1f3g-curso`. I don't know if the *current organization* is the organization the user is trying to join.